task/01-strings-tasks.js

```diff
@@ -236,12 +261,11 @@ function encodeToRot13(str) {
  *   isString({}) => false
  *   isString('test') => true
  *   isString(new String('test')) => true
- */
+ */ 
 function isString(value) {
-    throw new Error('Not implemented');
+       return (typeof(value) === 'string' || value instanceof String) ? true : false;
```

Comment:
Ternary is redundant here
Please, just return `(typeof(value) === 'string' || value instanceof String)`

Please, see https://javascript.info/logical-operators